cli/cmd/install-cni-plugin/template.go

```diff
@@ -0,0 +1,194 @@
+package installcniplugin
```

Comment:
My preference is to have this template live alongside the install template. Can you move this file to `cli/install/cni-template.go`? Since there's already a `Template` const in that package, let's call the const in this file `CNITemplate`.